### PR TITLE
#11570 Add a basic export to the environment and environment sample d…

### LINKED ISSUE #11570: Basic export of Env entities
### Feature Description

This issue is part of the EPIC #11533 on the environmental surveillance module.
### Proposed Change

Add a basic export of only what is shown on UI in .CSV, just as in other repositories

### PR DESCRIPTION
**Pull request by leventegal-she**
…irectories

Fixes #11570

### CODE REVIEW

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=12540

File: sormas-ui/src/main/java/de/symeda/sormas/ui/utils/ExportEntityName.java [hunk 1]
```diff
@@ -39,7 +39,9 @@ public enum ExportEntityName {
 	DOCUMENTS("entityDocuments", "documents"),
 	DATA_DICTIONARY("entityDataDictionary", "data dictionary"),
 	DATA_PROTECTION_DICTIONARY("entityDataProtectionDictionary", "data protection dictionary"),
-	PERSONS("entityPersons", "persons");
+	PERSONS("entityPersons", "persons"),
+	ENVIRONMENTS("entityEnvironments", "environments"),
+	ENVIRONMeNT_SAMPLES("entityEnvironmentSamples", "environment samples"),;
```
Comment (MateStrysewske) [inline sormas-ui/src/main/java/de/symeda/sormas/ui/utils/ExportEntityName.java:44]:
```suggestion
	ENVIRONMENT_SAMPLES("entityEnvironmentSamples", "environment samples");
```

**Review by MateStrysewske — CHANGES_REQUESTED** [review]

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=12540

File: sormas-ui/src/main/java/de/symeda/sormas/ui/utils/ExportEntityName.java [hunk 2]
```diff
@@ -39,7 +39,9 @@ public enum ExportEntityName {
 	DOCUMENTS("entityDocuments", "documents"),
 	DATA_DICTIONARY("entityDataDictionary", "data dictionary"),
 	DATA_PROTECTION_DICTIONARY("entityDataProtectionDictionary", "data protection dictionary"),
-	PERSONS("entityPersons", "persons");
+	PERSONS("entityPersons", "persons"),
+	ENVIRONMENTS("entityEnvironments", "environments"),
+	ENVIRONMENT_SAMPLES("entityEnvironmentSamples", "environment samples"),;
```
Comment (MateStrysewske) [inline sormas-ui/src/main/java/de/symeda/sormas/ui/utils/ExportEntityName.java:44]: Remove the comma here as well, please; you can merge the PR afterwards

**Review by MateStrysewske — CHANGES_REQUESTED** [review]

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=12540

**Pull request merged**